tasks/oss.yml

```diff
@@ -34,7 +34,7 @@
         path: "{{ oss_path }}"
         state: unmounted
       when:
-        - oss_dev is defined
+        - oss_dev != ''
```

Comment:
Sure this works? Sometimes it becomes an empty string, and other times it remains as None, for which you need `oss_dev is not none`

Reply:
Yes, because oss_dev is a variable defined at an upper scope and it resolved to `''` when oss_item.dev is not defined.

Reply:
Although, I have pushed a changed unrelated to this PR, I will open a new one.